components/ServiceCard/index.js

```diff
@@ -19,7 +21,14 @@ export default function ServiceCard() {
       </h2>
     );
   }
-
+  async function onDelete(id) {
+    if (!confirm("Are you sure you want to delete this service?")) {
+      return;
+    }
```

Comment:
I recommend using a modal instead of the default confirm dialog.

Reply:
We will work this into a later user story!